### PR TITLE
Pop-up to edit Surgery for weekly view of OT schedule

### PR DESCRIPTION
**Pull request by shivarach**
https://msfprojects.atlassian.net/browse/MOBN-1161

### CODE REVIEW

File: ui/app/ot/views/moveAppointment.html [hunk 1]
```diff
@@ -15,6 +15,10 @@ <h4> {{'Moving ' + surgicalAppointment.patient.display + ' from ' + sourceSurgic
         <i class="fa fa-exclamation-circle small"></i>
         <span>{{'NO_FREE_SLOT_AVAILABLE_FOR_THE_GIVEN_SURGEON_AND_DATE_MESSAGE' | translate}}</span>
     </p>
-    <button ng-mousedown="cancel()">{{'OT_SURGICAL_APPOINTMENT_CANCEL' | translate}}</button>
-    <button ng-click="moveSurgicalAppointment()" ng-disabled="!dateForMovingSurgery || !destinationBlock"> {{'OT_SURGICAL_APPOINTMENT_MOVE' | translate}}</button>
+    <div class="center">
+        <button ng-click="moveSurgicalAppointment()" ng-disabled="!dateForMovingSurgery || !destinationBlock">
```
Comment (kirity-mahanty) [inline ui/app/ot/views/moveAppointment.html:19]: Any idea why are we using 2 diff directives for 2 buttons `move` and `cancel`?

File: ui/test/unit/ot/controller/calendarViewController.spec.js [hunk 2]
```diff
@@ -488,6 +488,23 @@ describe("calendarViewController", function () {
...
+    it("should open surgical appointment dialog on clicking surgical appointment", function () {
+        createController();
+        scope.view = 'Calendar';
+        let surgicalAppointment = {status: Bahmni.OT.Constants.scheduled};
+        scope.$emit("event:surgicalAppointmentSelect", surgicalAppointment, {surgicalAppointments: []});
+
+        expect(ngDialog.open).toHaveBeenCalledWith(jasmine.objectContaining(
+            {
+                template: 'views/surgicalAppointmentDialog.html',
+                className: 'ngdialog-theme-default ng-dialog-adt-popUp ot-dialog',
```
Comment (kirity-mahanty) [inline ui/test/unit/ot/controller/calendarViewController.spec.js:500]: we should remove `ng-dialog-adt-popUp` from the className as we are not using it.

File: ui/app/ot/views/moveAppointment.html (hunk 1 above)
Comment (shivarach) [reply ui/app/ot/views/moveAppointment.html:19]: I hope it is discussed.

File: ui/test/unit/ot/controller/calendarViewController.spec.js (hunk 2 above)
Comment (shivarach) [reply ui/test/unit/ot/controller/calendarViewController.spec.js:500]: yes, will do.

Comment (shivarach) [reply ui/test/unit/ot/controller/calendarViewController.spec.js]: done

**Pull request merged**